libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -162,7 +177,6 @@ void HarmonicNotchFilter<T>::init(float sample_freq_hz, float center_freq_hz, fl
     NotchFilter<T>::calculate_A_and_Q(center_freq_hz, bandwidth_hz / _composite_notches, attenuation_dB, _A, _Q);
 
     _initialised = true;
-    update(center_freq_hz);
```

Comment:
why lose this? Pretty sure this was added to fix a bug

Reply:
because during init() we are just booting and motors are off. They are not running at center_freq_hz, zero is correct which is what it defaults to. We could do update(0) here if you like, though it won't make any difference